iaso/api/storage.py

```diff
@@ -193,27 +246,37 @@ def list(self, request):
         page_offset = request.GET.get("page", 1)
         order = request.GET.get("order", "updated_at,id").split(",")
 
+        csv_format = request.GET.get("csv", None)
+        xlsx_format = request.GET.get("xlsx", None)
+        file_export = False
+        if csv_format is not None or xlsx_format is not None:
```

Comment:
It's not very readable, something like this is more readable : 
```python

csv_format = request.GET.get("csv", None)
xlsx_format = request.GET.get("xlsx", None)
file_export = False

if csv_format is not None :
     file_export = True
     file_format_export = FileFormatEnum.CSV 
if xlsx_format is not None : 
     file_export = True
     file_format_export = FileFormatEnum.XLSX
```

(Not really important but ...)

Reply:
On pourrait aussi faire if csv_format ... elif xlsx_format ...

Reply:
Indeed, thanks for the suggestion, I just made the change. The backend code could be improved further if the frontend gave a single parameter (`export=xlsx` for example), since it doesn't make sense to request both a CSV and XLSX export in the same request, but I kept things like that for consistency with the rest of Iaso.